core/src/main/java/com/confighub/core/utils/FileUtils.java

```diff
@@ -71,15 +76,8 @@ public static String resolveFile(final Context context,
             file.decryptFile(pass);
         }
 
-        Map<String, Property> propertyMap = new HashMap<>();
-        Map<String, PropertyKey> keyMap = new HashMap<>();
-        resolved.keySet().forEach(k -> {
-            propertyMap.put(k.getKey(), resolved.get(k));
-            keyMap.put(k.getKey(), k);
-        });
-
         String fileContent = file.getContent();
-        String patternString = "(?i)\\$\\{\\s*\\b(" + StringUtils.join(propertyMap.keySet(), "|") + ")\\b\\s*}";
+        String patternString = "(?i)\\$\\{\\s*\\b([\\w\\.]+)\\b\\s*}";
```

Comment:
Are all properties words in regex form?  Any characters that confighub allows but \w in regex doesn't?

Reply:
Good catch. I was searching for A-Za-z0-9_. but we also allow []()*+-